src/commands/fatlog-get.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Although I understand the rational behind a new command, a fat command is logically different from a slow command, I would also like us to more seriously consider just adding one new field to the slowlog, the output/input bytes, and then having both get logged to the same place. It's not a perfect fit, but then users can just look in "one" place for bad behavior. It also allows the two logs to perhaps explain each other, since a large output can cause a slow command. We could extend `SLOWLOG GET [count] [TYPE FAT|SLOW]` to only show entries of the specific type. 

I'm thinking about it more from a simpler way to explain it to end users.

Reply:
This does make things easier than adding a new series of commands, if we can ignore that changes in format of command reply may cause backwards compatibility problem.